src/GtkUtilities.jl

```diff
@@ -44,7 +44,7 @@ include("panzoom.jl")
 using .PanZoom
 import .PanZoom: interior, fullview   # for extensions
 
-function Base.copy!{C<:Colorant}(ctx::CairoContext, img::AbstractArray{C})
+function Base.copy!{C<:Union{Colorant,Number}}(ctx::CairoContext, img::AbstractArray{C})
```

Comment:
When I've asked in the past, folks have wanted to keep the wrappers "minimalistic" and faithful to the Gtk/Cairo API. Otherwise this wouldn't even be a separate package.